### ISSUE TITLE
Readme Question

### ISSUE THREAD

**Issue by jzhanglsw** [rousez/Auto-Amazon-Giveaways#16]
This may be a really stupid question but is this a script to automatically enter Amazon giveaways? It's not immediately obvious from the readme what this does.

**Comment by sniffingpickles** [comment]
Yes, it automatically enters the non-requirement giveaways.  I've updated my fork to enter the "video watching" giveaways, but overall this just cycles through the pages and enters what it sees.

**Comment by jzhanglsw** [comment]
Great work. I'm definitely going to check this out later.
